app/models/ability.rb

```diff
@@ -5,6 +5,7 @@ def initialize(user)
     if user
       can %i(create read destroy update), Wishlist, user_id: user.id
       can %i(create update), Item, wishlist: { user_id: user.id }
+      can %i(destroy), Reservation
```

Comment:
I think it's also **really important** for this to be tested. When we allow someone to delete something, we should first and foremost test if they are not deleting something they shouldn't.